### PR TITLE
missing osqueryi to build hubble package.

### PR DESCRIPTION
**Pull request by zachsis**
ref: https://github.com/hubblestack/hubble/issues/895 

This resolved building a custom hubble package for me.

### CODE REVIEW

File: pkg/osquery/Dockerfile [hunk 1]
```diff
@@ -48,6 +48,7 @@ USER root
 #collect all files and prepare them into a tar file
 RUN mkdir -p /opt/osquery/lenses /opt/osquery/yara /opt/osquery/extensions \
  && cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryd /opt/osquery \
```
Comment (MoodyMudit) [inline pkg/osquery/Dockerfile:50]: I know this is a separate issue altogether. However, osqueryd needs to be copied as **/opt/osquery/hubble_osqueryd**. Refer [here](https://github.com/hubblestack/hubble/blob/develop/hubblestack/extmods/modules/nebula_osquery.py#L1291)

**Comment by MoodyMudit** [comment]
@jettero This error is a recent one, when the dockerfiles for osquery and hubble were separated.

File: pkg/osquery/Dockerfile [hunk 2]
```diff
@@ -48,6 +48,7 @@ USER root
...
+ && cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryi /opt/osquery/ \
```
Comment (MoodyMudit) [inline pkg/osquery/Dockerfile:51]: Also **/home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryi** doesn't actually exists. The osqueryd binary is the same as osqueryi binary. So osqueryd needs to be copied at both places. The correct cp statement should be **cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryd /opt/osquery/osqueryi**

**Comment by MoodyMudit** [comment]
Closing this one since I have opened a new PR with fixes https://github.com/hubblestack/hubble/pull/899

File: pkg/osquery/Dockerfile (hunk 2 above)
Comment (jettero) [reply pkg/osquery/Dockerfile:51]: or maybe a hardlink? if it's really the same file, symlinking or hardlinking would seem the right thing to do to me.

**Pull request closed without merging**